doc/source/development/contributing_gitpod.rst

```diff
@@ -0,0 +1,277 @@
+
+Gitpod
+------
+
+`Gitpod`_  is an open-source platform for automated and ready-to-code
+development environments. It enables developers to describe their dev
+environment as code and start instant and fresh development environments for
+each new task directly from your browser. This reduces the need to install local
+development environments and deal with incompatible dependencies.
+
+
+Gitpod GitHub integration
+-------------------------
+
+To be able to use Gitpod, you will need to have the Gitpod app installed on your
+GitHub account, so if
+you do not have an account yet, you will need to create one first.
+
+To get started just login at gitpod, and grant the appropriate permissions to github.
+
+We have built a python 3.8 environment and all development dependencies will
+install when the environment starts.
+
+
+Forking the pandas repository
+-----------------------------
+
+The best way to work on pandas as a contributor is by making a fork of the
+repository first.
+
+#. Browse to the `pandas repository on GitHub`_ and `create your own fork`_.
+
+#. Browse to your fork. Your fork will have a URL like
+   https://github.com/noatamir/pandas-dev, except with your GitHub username in place of
+   ``noatamir``.
+
+Starting Gitpod
+---------------
+Once you have authenticated to Gitpod through GitHub, you can install the
+`Gitpod browser extension <https://www.gitpod.io/docs/browser-extension>`_
+which will add a **Gitpod** button next to the **Code** button in the
+repository:
+
+.. image:: ./gitpod-imgs/pandas-github.png
+   :alt: pandas repository with Gitpod button screenshot
+
+#. If you install the extension - you can click the **Gitpod** button to start
+   a new workspace.
+
+#. Alternatively, if you do not want to install the browser extension, you can
+   visit https://gitpod.io/#https://github.com/USERNAME/pandas replacing
+   ``USERNAME`` with your GitHub username.
+
+#. In both cases, this will open a new tab on your web browser and start
+   building your development environment. Please note this can take a few
+   minutes.
+
+#. Once the build is complete, you will be directed to your workspace,
+   including the VSCode editor and all the dependencies you need to work on
+   pandas. The first time you start your workspace, you will notice that there
+   might be some actions running. This will ensure that you have a development
+   version of pandas installed.
+
+#. When your workspace is ready, you can :ref:`test the build<contributing.running_tests>` by
+   entering::
+
+      $ python pytest pandas
+
+
+Quick workspace tour
+--------------------
+Gitpod uses VSCode as the editor. If you have not used this editor before, you
+can check the Getting started `VSCode docs`_ to familiarize yourself with it.
+
+Your workspace will look similar to the image below:
+
+.. image:: ./gitpod-imgs/gitpod-workspace.png
+   :alt: Gitpod workspace screenshot
+
+.. note::  By default, VSCode initializes with a light theme. You can change to
```

Comment:
Somehow mine did not, installed it fresh to try gitpod. Got the dark theme right from the beginning